README.md

```diff
@@ -87,6 +87,20 @@ where:
 
 The validator can then be executed via bash commands. See the [preceeding instructions for command line usage](#run-the-app-via-command-line).
 
+### Build Docker container locally
```

Comment:
I think we could add this in the "### Setup section". We would have the subsections "#### Build Docker container locally" and "#### Pull the latest docker image". The current step "1. Download and install Docker" should also be added to this.